Kitodo-API/src/main/java/org/kitodo/config/Config.java

```diff
@@ -27,6 +29,17 @@
 
 public class Config {
     private static final Logger logger = LogManager.getLogger(Config.class);
+
+    private static final Field FIELD_DETAIL_MESSAGE;
```

Comment:
Is it necessary to take a sledgehammer to crack a nut?

Reply:
Our discoussion showd that the previous approach istn't what any of us wanted. Rewriting the exception message seems the best solution for it. It preserves the original stacktrace which is valuable in case of working with a debugger; how ever we do not need to catch and log it in order to throw a new exception.


Sorry for the necessity to handle all those checked exceptions, but in my eyes that's Java's bad API design.